src/devices/bus/isa/svga_trident.cpp

```diff
@@ -60,9 +65,8 @@ void isa16_svga_tvga9000_device::device_start()
 
 	m_isa->install_rom(this, 0xc0000, 0xc7fff, "tvga9000");
 
-	m_isa->install_device(0x3b0, 0x3bf, read8sm_delegate(*m_vga, FUNC(tvga9000_device::port_03b0_r)), write8sm_delegate(*m_vga, FUNC(tvga9000_device::port_03b0_w)));
-	m_isa->install_device(0x3c0, 0x3cf, read8sm_delegate(*m_vga, FUNC(tvga9000_device::port_03c0_r)), write8sm_delegate(*m_vga, FUNC(tvga9000_device::port_03c0_w)));
-	m_isa->install_device(0x3d0, 0x3df, read8sm_delegate(*m_vga, FUNC(tvga9000_device::port_03d0_r)), write8sm_delegate(*m_vga, FUNC(tvga9000_device::port_03d0_w)));
+	m_isa->install_device(0x03b0, 0x03df, *this, &isa16_svga_tvga9000_device::io_isa_map);
```

Comment:
Notice that ISA bus seems to dislike a logical mapping of
`m_isa->install_device(0x03b0, 0x03df, *m_vga, &tvga9000_device::io_map);`
at compile time. Not that I mind too much, I'm of the opinion that all these read8sm delegates collections are hard to read plus an ISA VGA card can eventually have a specific sub-interface in the bus slot (particularly true if/when we ever need to support VLB + the actual mechanism for video subsystem enable).